websites/M/MyTF1/presence.ts

```diff
@@ -0,0 +1,168 @@
+const presence = new Presence({
+	clientId: "1043951049800695808",
+}),
+strings = presence.getStrings({
+	viewHome: "general.viewHome",
+	viewPage: "general.viewPage",
+	viewProfile: "general.viewProfile",
+	watchingVid: "general.watchingVid",
+	browse: "general.browsing",
+	live: "general.live"
+}),
+browsingTimestamp = Math.floor(Date.now() / 1000);
+
+presence.on("UpdateData", async () => {
+	
+	const presenceData: PresenceData = {
+		largeImageKey: "mytf1",
+		startTimestamp: browsingTimestamp,
+	},
+	path = window.location.pathname,
```

Comment:
`document` is preferred in this repository.
```suggestion
	path = document.location.pathname,
```